Formula/universal-ctags.rb

```diff
@@ -0,0 +1,57 @@
+class UniversalCtags < Formula
+  desc "Maintained ctags implementation"
+  homepage "https://github.com/universal-ctags/ctags"
+  url "https://github.com/universal-ctags/ctags/archive/refs/tags/p5.9.20210523.0.tar.gz"
+  sha256 "00e5493454c7c013b996a24019b8f8cbd75968e1993fb56f7e2737b7309cc684"
+  license "GPL-2.0-only"
+  head "https://github.com/universal-ctags/ctags.git"
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "docutils" => :build
+  depends_on "pkg-config" => :build
+  depends_on "jansson"
+  depends_on "libyaml"
+  uses_from_macos "libxml2"
+  conflicts_with "ctags", because: "this formula installs the same executable as the ctags formula"
```

Comment:
```suggestion
  depends_on "libyaml"

  uses_from_macos "libxml2"

  conflicts_with "ctags", because: "this formula installs the same executable as the ctags formula"
```
Same.